editor/src/components/canvas/controls/size-box.tsx

```diff
@@ -111,6 +118,7 @@ class ResizeControl extends React.Component<ResizeControlProps> {
         ) : (
           this.props.children
         )}
+        {/*
```

Comment:
Is this intentional? If so, should it be deleted?

Reply:
Ah yes, that should probably be deleted.

Reply:
Fixed in 5b68dd64a689f218fbe4e203d4db7b5a9433a636.